packages/smart-vaults/src/utils/MultiSigner.sol

```diff
@@ -168,27 +169,24 @@ abstract contract MultiSigner {
 
         MultiSignerLib.MultiSignerStorage storage $ = _getMultiSignerStorage();
 
-        bytes memory signer;
-
         for (uint8 i; i < numSigners; i++) {
-            signer = signers_[i];
-            MultiSignerLib.validateSigner(signer);
-            $.signers[i] = signer;
+            MultiSignerLib.validateSigner(signers_[i]);
+            $.signers[i] = signers_[i];
```

Comment:
technically could save an SSTORE for EOAs, not sure if worth it tho

Reply:
actually just having some kind of `set` or `save` fn on the storage pointer should be easy/clean and save the gas

Reply:
saving around 100 gas, not worth it imo

Reply:
oh wait, in initialize we could save 2200, hmm

Reply:
yeah it should be pretty easy so i think it's worth doing